app/filtering.js

```diff
@@ -235,31 +236,31 @@ function registerForHeadersReceived (session) {
 function registerPermissionHandler (session, partition) {
   const isPrivate = !partition.startsWith('persist:') && partition !== '' && partition !== 'main-1'
   // Keep track of per-site permissions granted for this session.
-  // TODO: Localize strings
-  let permissions = {
-    media: {
-      action: 'use your camera and/or microphone'
-    },
-    geolocation: {
-      action: 'see your location'
-    },
-    notifications: {
-      action: 'show you notifications'
-    },
-    midiSysex: {
-      action: 'use web MIDI'
-    },
-    pointerLock: {
-      action: 'disable your mouse cursor'
-    },
-    fullscreen: {
-      action: 'use fullscreen mode'
-    },
-    openExternal: {
-      action: 'open an external application'
-    }
-  }
+  let permissions = {}
   session.setPermissionRequestHandler((webContents, permission, cb) => {
+    permissions = {
```

Comment:
prefer to only set this once if not already set, because it will be a callback called a lot.